spec/swagger_helper.rb

```diff
@@ -1,13 +1,13 @@
 require 'rails_helper'
 
-Dir[::Rails.root.join("spec/swagger/**/schemas/*.rb")].each { |f| require f }
-Dir[::Rails.root.join("spec/swagger/**/definition.rb")].each { |f| require f }
+Dir[::Rails.root.join("spec/swagger/**/schemas/*.rb")].sort.each { |f| require f }
+Dir[::Rails.root.join("spec/swagger/**/definition.rb")].sort.each { |f| require f }
 
 RSpec.configure do |config|
   # Specify a root folder where Swagger JSON files are generated
   # NOTE: If you're using the rswag-api to serve API descriptions, you'll need
   # to ensure that it's confiugred to serve Swagger from the same folder
-  config.swagger_root = Rails.root.to_s + '/swagger'
+  config.swagger_root = "#{Rails.root}/swagger"
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Rails/FilePath: Please use Rails.root.join('path/to') instead.